integration/Makefile

```diff
@@ -1,13 +1,24 @@
-test: gen_proto servers clients
+test: gen_proto gradlew_install go_servers java_servers go_clients java_client
 
 gen_proto:
-	protoc -I proto/ proto/defs.proto --go_out=plugins=grpc:proto
+	protoc -I src/main/proto/ src/main/proto/defs.proto --go_out=plugins=grpc:proto
 
-servers:
+gradlew_install:
+	./gradlew installDist
+
+go_servers:
 	# TODO: Capture these PIDs and kill them
 	go run go-grpc-server/main.go &
 	go run go-http-server/main.go &
 
-clients:
+java_servers:
+	# TODO: Capture these PIDs and kill them
+	build/install/java-interop/bin/GrpcInteropTestServer &
+
+go_clients:
 	go test -v go-grpc-client/*.go
 	go test -v go-http-client/*.go
+
+java_client:
+	build/install/java-interop/bin/GrpcInteropTestClient 9800
```

Comment:
Could we put these in environment variables and iterate in the app itself, like we do for the Go version?

Reply:
SG, done.